dhall/tests/format/letShortCommentsB.dhall

```diff
@@ -1 +1,7 @@
-let x = True let {- 0 -} y {- 1 -} : {- 2 -} Bool = {- 3-} False in x
\ No newline at end of file
+let x = True
+
+let {- 0 -} y
+    {- 1 -} : {- 2 -} Bool
```

Comment:
It might look nicer to move the `{- 1 -}` to the line above. no? IMHO the rest of the line after the variable is the best position for a comment.

Reply:
Yeah, I agree

Reply:
The handling of line comments is already pretty reasonable though. I might come back to this later.